Tools/scripts/install-travis-env.sh

```diff
@@ -19,8 +19,8 @@ UBUNTU64_PKGS="libc6:i386 libgcc1:i386 gcc-4.6-base:i386 libstdc++5:i386 libstdc
 
 # GNU Tools for ARM Embedded Processors
 # (see https://launchpad.net/gcc-arm-embedded/)
-ARM_ROOT="gcc-arm-none-eabi-4_7-2014q2"
-ARM_TARBALL="$ARM_ROOT-20140408-linux.tar.bz2"
+ARM_ROOT="gcc-arm-none-eabi-4_9-2015q3"
+ARM_TARBALL="$ARM_ROOT-20150921-linux.tar.bz2"
```

Comment:
It's needed for px4-v2 compilation, previous version there isn't available on http request. 

Please check, this is the previous declared gcc tools link for download, but is not available anymore, follow the link to confirm absence:

http://firmware.diydrones.com/Tools/PX4-tools/gcc-arm-none-eabi-4_7-2014q2-20140408-linux.tar.bz2